arex-schedule-web-api/src/main/java/com/arextest/schedule/service/PlanConsumePrepareService.java

```diff
@@ -155,4 +177,61 @@ private int doFixedCaseSave(ReplayActionItem replayActionItem) {
         replayActionCaseItemRepository.save(caseItemList);
         return size;
     }
+
+
+    public boolean prepareAndUpdateFailedActionAndCase(ReplayPlan replayPlan) {
```

Comment:
Consider memory issues and avoid full field spit out, such as Target request

Reply:
fixed